CMakeLists.txt

```diff
@@ -1,224 +1,224 @@
-################################################################################
```

Comment:
What happened to using cmake variables to selectively build musica or musica fortran? Did we decide that's not a good idea?

Reply:
Preserving the usability of musica with fetch content will not work without a top level cmake file

Reply:
If it's too messy to separate them cleanly maybe we should just create yet another repository